### DISCUSSION TITLE
Add aria attribute to emojis

### DISCUSSION THREAD

**Discussion by Samiam519** [fomantic/Fomantic-UI#1325] [Elements (button, image, label, loader, ...)] [answered]
Currently emoji characters do not provide any labels for keyboard users. Is there a way to add an `aria-label` attribute similar to FontAwesome icons?

**Comment by lubber-de** [comment]
Sure, just add the aria label to either the emoji itself or surround it by a separate tag. Just as you would do with the icons
```html
<a aria-label="Delete" class="btn btn-danger" href="path/to/settings">
  <em aria-hidden="true" data-emoji=":sad:" title="Delete this item?"></em>
</a>
```

**Comment by Samiam519** [comment]
FontAwesome icons already have a built in aria-label is my point. I was hoping for this support to be added without having to manually add a label for each character.

**Comment by lubber-de** [comment, marked as answer]
Fomantic does currently only provide the pure CSS for emojis tags and no js module for html tag creation, so you would have to add the desired aria labels as html attributes by yourself.
